src/app/app-routing.module.ts

```diff
@@ -10,14 +11,14 @@ const routes: Routes = [
     children: [
       {
         path: '',
-        data: { title: 'Home' },
-        canActivate: [MsalGuard],
+        data: { title: 'Home', roles: ['CVSFullAccess', 'TechRecord.Read'] },
```

Comment:
This'll be TechRecord.View not read

Reply:
Thanks, this is honestly just placeholder for now as none of it even exits in AD yet haha.